### ISSUE TITLE
Request for pre-trained VEN / Occlusion Detector weights

### ISSUE THREAD

**Issue by 2836317064-ops** [Ayush-00/mimicgait#2]
Hi @Ayush-00,

Thank you for releasing the code of **MimicGait**! The paper and implementation look very interesting.

I am trying to reproduce the VEN-GaitGL (or Mimic network) results on the GREW dataset. However, when I follow the instructions to train the full pipeline, I encounter the following issue:

When loading the `occ_detector_path` in `gaitgl_occ_aware.py`, I get a **state_dict key mismatch error**:
- Missing keys: `layer1.0.weight`, `layer1.0.bias`, `layer2.0.weight`, ..., `layer4.0.weight`, etc.
- Unexpected keys: `weight`, `bias`

This happens because the checkpoint I provide only contains simple `weight`/`bias` keys, while the `Occ_Detector` expects the full multi-layer CNN structure.

According to the README, we need to first train the VEN using:
```bash
cd occlusion_detector
python train.py --dataset grew
and then use the saved best-model.pth from the saves/ folder.
Training VEN from scratch requires the full preprocessed GREW dataset and takes additional time and GPU resources.
Could you please release the pre-trained VEN weights (especially best-model.pth or the final checkpoint) so that the community can more easily reproduce the main MimicGait results?
It would be very helpful if you could upload it to:

GitHub Releases, or
A Google Drive / OneDrive / Hugging Face link, or
Any other public link

If releasing the weights is not possible at the moment, any guidance on the exact training settings (epochs, learning rate, batch size, data preprocessing details, etc.) that produce a compatible checkpoint would also be appreciated.
Thank you very much for your time and for this great work!
Best regards,
